tests/test_wires.py

```diff
@@ -318,8 +318,7 @@ def test_all_wires_method(self):
         new_wires = Wires.all_wires([wires1, wires2, wires3], sort=True)
         assert new_wires.labels == (1, 2, 3, 4, 5, 6)
 
-        with pytest.raises(WireError, match="Expected a Wires object"):
-            Wires.all_wires([[3, 4], [8, 5]])
+        assert Wires.all_wires([[3, 4], [8, 5]]).labels == (3, 4, 8, 5)
```

Comment:
all_wires no longer raises when it is given something that can be cast to a `Wires` object.